### DISCUSSION TITLE
黑群晖Drive文档服务穿透，远程主机开始同步文档，就不断重试连接？

### DISCUSSION THREAD

**Discussion by CoffeeRong** [natfrp/wiki#477] [使用问题] [answered]
尝试了永隧道穿透黑群晖的Drive文档服务，远程客户端设置了对应的地址，可以连上服务。但是点击开始同步文档后，日志不断弹出重试连接。暂停同步后，现象没有好转。尝试迁移节点，也是会在开始同步后出现重试连接。已经迷糊了，希望好心人帮忙解决
![PixPin_2024-05-11_11-10-36](https://github.com/natfrp/wiki/assets/45674719/2a5db604-802c-4a7e-a4a9-f738dceb6df8)
![PixPin_2024-05-11_11-09-17](https://github.com/natfrp/wiki/assets/45674719/48468c31-e20e-46ca-95ee-dc18d5900fc5)
[SakuraLauncher_20240511_110255.log](https://github.com/natfrp/wiki/files/15281459/SakuraLauncher_20240511_110255.log)

**Comment by fxy1699** [comment]
同上，用SFTP传输文件之后就会重连

**Comment by CoffeeRong** [comment, marked as answer]
看了一下QQ群，说是服务端更新出的问题，现在回滚更新后。Drive又能同步了
